### PR TITLE
feat: Add Carousel component desktop and mobile responsive in Event details page

### PR DESCRIPTION
**Pull request by SilviaTrujillano23**
## Issue ticket number and link

https://trello.com/c/KDuxzkuj

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] If it is a core feature, I have added thorough tests.
- [ ] Do we need to implement analytics?
- [ ] Will this be part of a product update? If yes, please write one phrase about this update.

## Pull request type

Please check the type of change your PR introduces:

- [x] :sparkles: Feature
- [x] :bug: Bugfix
- [ ] :pencil2: Fixing typos
- [ ] :lipstick: :art: Code style update (formatting, renaming)
- [ ] :recycle: Refactoring (no functional changes, no api changes)
- [ ] :zap: Performance
- [ ] :pencil: Documentation content changes
- [ ] :white_check_mark: Unit testing
- [ ] :ok_hand: Updating due to code review suggest
- [ ] :rocket: Release: v
- [ ] :wrench: Add or update configuration files
- [ ] :hammer: Add or update development scripts
- [ ] :bricks: Infrastructure related changes

## What is the current module?

- Carousel component in Event details page
- Endpoint to get event details pointing not-functional endpoint (TODO for next sprint)

## What is the new behavior?

- Implemented Carousel component responsive for desktop and mobile devices

Desktop version

image
image

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: src/shared/api/endpoints/events.ts [hunk 1]
```diff
@@ -19,7 +19,9 @@ function Events() {
   }
 
   const getAllEventsAuth = async () => {
-    const { data } = await requests.get(`${APIEndpoints.users}/${APIEndpoints.events}`)
+    //TODO: Add atendees to the request in endpoint
+    // const { data } = await requests.get(`${APIEndpoints.users}/${APIEndpoints.events}`)
```
Comment (jlora-mck) [inline src/shared/api/endpoints/events.ts:23]: Please remove this

File: package.json [hunk 2]
```diff
@@ -24,6 +24,7 @@
     "react-i18next": "^14.0.1",
     "react-router-dom": "^5.2.0",
     "react-scripts": "^4.0.3",
+    "react-swipeable-views": "^0.14.0",
```
Comment (jlora-mck) [inline package.json:27]: Are we using this ?

File: src/shared/api/endpoints/events.ts (hunk 1 above)
Comment (SilviaTrujillano23) [reply src/shared/api/endpoints/events.ts:23]: Will do, but pointing to this endpoint is temporary until BE team fix and build a new one.

File: package.json (hunk 2 above)
Comment (SilviaTrujillano23) [reply package.json:27]: Removed

File: src/components/Carousel/Carousel.tsx [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+export interface CarouselProps {
+  items: { img: string; title: string }[]
+}
+
+const Carousel: React.FC<CarouselProps> = ({ items }) => {
+  const classes = carouselStyles()
+  const theme = useTheme()
+  const isMobile = useMediaQuery(theme.breakpoints.down('sm'))
+
+  const [activeStep, setActiveStep] = useState(0)
```
Comment (jlora-mck) [inline src/components/Carousel/Carousel.tsx:19]: please type this state

File: src/components/Carousel/Carousel.tsx [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+
+  const handleNext = () => {
+    setActiveStep((prevActiveStep) => prevActiveStep + 1)
+  }
+
+  const handleBack = () => {
+    setActiveStep((prevActiveStep) => prevActiveStep - 1)
+  }
+  const maxDesktopSteps = 3
```
Comment (jlora-mck) [inline src/components/Carousel/Carousel.tsx:28]: add this const outside the function and with all caps something 

const MAX_DESKTOP_STEPS = 3

**Review by jlora-mck — APPROVED** [review]

File: src/components/Carousel/Carousel.tsx (hunk 3 above)
Comment (SilviaTrujillano23) [reply src/components/Carousel/Carousel.tsx]: done

File: src/components/Carousel/Carousel.tsx (hunk 4 above)
Comment (SilviaTrujillano23) [reply src/components/Carousel/Carousel.tsx]: Moved to constants

**Review by milca19 — APPROVED** [review]

**Pull request merged**